docs/competition/teams/role-descriptions/reception.md

```diff
@@ -9,49 +9,53 @@ original:
 ---
 # Reception Volunteer
 
-Reception staff control entry to the building and validate people's competition
-tickets and/or wristbands. Each reception staffer has a desk, chair, and
-computer.
+Reception staff control entry to the building and validate people's Media Consent Forms (MCFs) and/or wristbands. Each reception staffer has their own desk and chair.
 
-There are four flavours of visitor to the competition: competitors,
-team-leaders, volunteers and true-visitors. **Everyone** must complete a media consent form (MCF) OR hold a valid ticket (which also counts as an MCF). Under 16's must have this form completed by a parent or guardian. If someone denies Media Consent, or is under age and doesn't have a form, please speak to the volunteer responsible for Managing Teams.
+Here is a flow chart summarising the details below:
 
-When someone enters the building, you should follow the procedure below.
+![Reception Desk Flow](../diagrams/reception-desk-flow.svg)
 
-**Note: Training on how to use the scanning software will be given on the morning by the competition team coordinator.**
+**Everyone** must complete a Media Consent Form, Under 16's must have this form completed by a parent or guardian. If they have signed a form, or if they bring a pre-signed form, give them a wristband and ensure you see them put it on. Also, give each competitor a competition leaflet.
 
-![Reception Desk Flow](../diagrams/reception-desk-flow.svg)
+Please strongly dissuade them from rejecting media consent, as it causes a lot of pains with regards to taking photos of the event. However, they do have the right to reject it. In that case, give them a red lanyard to wear the entire time instead of their wristband. Remember to give them a leaflet too.
```

Comment:
```suggestion
If they refuse, we should double check they understand what they are refusing. Media consent means that we can take photos or videos with them in and use this media to advertise ourselves. We should explain that photos and videos are being taken around the event and that it would be best for them to avoid the arena as we are also livestreaming. If they still refuse media consent provide them with a red lanyard to wear **instead** of a wristband. Instruct them they must wear this lanyard at all times around the event. Remember to give them a booklet too.
```